resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/utils/CircuitBreakerUtil.java

```diff
@@ -0,0 +1,20 @@
+package io.github.resilience4j.circuitbreaker.utils;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreaker.State;
+
+import static io.github.resilience4j.circuitbreaker.CircuitBreaker.State.*;
+
+public final class CircuitBreakerUtil {
+
+    /**
+     * Indicates whether Circuit Breaker allows any calls or not.
+     *
+     * @param circuitBreaker to test
+     * @return call is permitted
+     */
+    public static boolean isCallPermitted(CircuitBreaker circuitBreaker) {
```

Comment:
Sry, I don't remember, why we need this method again 😕 
It's basically the same functionality like `circuitBreaker.tryAcquirePermission`, but without emitting an `CircuitBreakerOnCallNotPermittedEvent`. You want to check the state of a CircuitBreaker, but without producing an event, correct?